cpp/csv_utils_module/algorithm/csv_utils.cpp

```diff
@@ -0,0 +1,51 @@
+#include "csv_utils.hpp"
+
+namespace CsvUtils {
+
+void CreateCsvFile(mgp_list *args, mgp_graph *memgraph_graph, mgp_result *result, mgp_memory *memory) {
+  mgp::MemoryDispatcherGuard guard(memory);
+  const auto arguments = mgp::List(args);
+  const auto record_factory = mgp::RecordFactory(result);
+
+  try {
+    const std::string_view filepath = arguments[0].ValueString();
+    const auto content = arguments[1].ValueString();
+    const auto isAppend = arguments[2].ValueBool();
```

Comment:
Use snake case for variable naming
```suggestion
    const auto is_append = arguments[2].ValueBool();
```